Project/app/src/main/java/org/ionproject/android/MainActivity.kt

```diff
@@ -74,69 +83,95 @@ class MainActivity : AppCompatActivity() {
      * This method is called by the framework after OnCreate but before it finishes
      */
     override fun onCreateOptionsMenu(menu: Menu?): Boolean {
-        menuInflater.inflate(R.menu.top_bar_menu, menu)
+        menuInflater.inflate(
+            R.menu.top_bar_menu,
+            menu
+        )
 
-        // Get the SearchView and set the searchable configuration
+        // Get the SearchView and set the searchable configuration.
         val searchManager = getSystemService(Context.SEARCH_SERVICE) as SearchManager
-        val menuSearchItem = menu?.findItem(R.id.action_search)
-        (menuSearchItem?.actionView as SearchView).apply {
+
+        val searchView = menu?.findItem(R.id.action_search)?.actionView as SearchView
```

Comment:
Still struggling with null safety... ;)
1. onCreateOptionsMenu argument is not nullable
2. the result of findItem is, though
This means that the line should read:
> menu?.findItem(R.id.action_search)?.actionView as? SearchView